README.md

```diff
@@ -12,7 +12,7 @@ The Mattermost Zoom integration allows team members to initiate a Zoom meeting w
 
 **Important**: Only Zoom users associated with the Zoom Account that created the Zoom App will be able to use the plugin. You can add these users from the **Manage Users** section in the Zoom Account settings.
 
-![image](https://github.com/mattermost/mattermost-plugin-zoom/assets/55234496/a109df21-f3c4-4432-b42d-7ef3ae493a50)
+![image](https://github.com/mattermost/mattermost-plugin-zoom/blob/master/assets/example.png)
```

Comment:
I think we'll want to test with relative paths instead of the absolute path. Also I suspect none of the URLs in any format will have that generated hash form involved

```suggestion
![image](./assets/example.png)
```